cargo/cargo_bootstrap.bzl

```diff
@@ -132,7 +132,12 @@ def _collect_environ(repository_ctx, host_triple):
             fail("File for key '{}' does not exist: {}", key, env_labels[key])
     env_labels = {key: str(value) for (key, value) in env_labels.items()}
 
-    return dict(env_vars.items() + env_labels.items())
+    if host_triple == "*":
```

Comment:
```suggestion
    # In addition to platform specific environment variables, a common set (indicated by `*`) will always
    # be gathered.
    if host_triple == "*":
```

Reply:
Maybe the use of `*` should also make it's way into user facing docs (maybe the attribute on `cargo_bootstrap_repository`)?